config/jobs/kubernetes/sig-testing/coverage.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+- interval: 1h
+  name: ci-kubernetes-coverage-conformance
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180829-29e661965-master
+      command:
+      - bash
+      args:
+      - -e
+      - -c
+      - |
+        shopt -s globstar
```

Comment:
oh this is uh.. ok

Reply:
Someone will regret this in the future :)

Reply:
yeah... 
Ideally we're going to replace this when we get everything out of bootstrap / scenarios / ... Sen and I would really like to get a better kubetest and better patterns for test scripts without any of the existing junk in the not terribly distant future... 

of course more likely we won't get to that and inline bash in pods will continue to be all over kubernetes testing /shrug